src/test/java/com/salessparrow/api/functional/controllers/crmOrganizationUserController/GetCrmOrganizationUserListTest.java

```diff
@@ -0,0 +1,122 @@
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.MethodSource;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.context.annotation.Import;
+import org.springframework.core.io.DefaultResourceLoader;
+import org.springframework.core.io.Resource;
+import org.springframework.http.MediaType;
+import org.springframework.test.context.web.WebAppConfiguration;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.ResultActions;
+import org.springframework.test.web.servlet.request.MockMvcRequestBuilders;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.github.dynamobee.exception.DynamobeeException;
+import com.salessparrow.api.helper.Cleanup;
+import com.salessparrow.api.helper.Common;
+import com.salessparrow.api.helper.FixtureData;
+import com.salessparrow.api.helper.LoadFixture;
+import com.salessparrow.api.helper.Scenario;
+import com.salessparrow.api.helper.Setup;
+import com.salessparrow.api.lib.globalConstants.CookieConstants;
+import com.salessparrow.api.lib.httpLib.HttpClient.HttpResponse;
+import com.salessparrow.api.lib.salesforce.helper.MakeCompositeRequest;
+
+import jakarta.servlet.http.Cookie;
+
+@SpringBootTest
+@AutoConfigureMockMvc
+@WebAppConfiguration
+@Import({ Setup.class, Cleanup.class, Common.class, LoadFixture.class })
+public class GetCrmOrganizationUserListTest {
+    @Autowired
+    private MockMvc mockMvc;
+    @Autowired
+    private Common common;
+    @Autowired
+    private LoadFixture loadFixture;
+    @Autowired
+    private Setup setup;
+    @Autowired
+    private Cleanup cleanup;
+
+    @MockBean
+    private MakeCompositeRequest makeCompositeRequestMock;
+
+    @BeforeEach
+    public void setUp() throws DynamobeeException {
+        setup.perform();
+    }
+
+    @AfterEach
+    public void tearDown() {
+        cleanup.perform();
+    }
+
+    @ParameterizedTest
+    @MethodSource("testScenariosProvider")
+    public void getCrmOrganizationUserList(Scenario testScenario) throws Exception {
+        // Load fixture data
+        String currentFunctionName = new Object(){}.getClass().getEnclosingMethod().getName();
+        FixtureData fixtureData = common.loadFixture("classpath:fixtures/controllers/crmOrganizationUserController/getCrmOrganizationUserList.fixtures.json", currentFunctionName);
+        loadFixture.perform(fixtureData);
+
+        // Read data from the scenario
+        ObjectMapper objectMapper = new ObjectMapper();
+
+        String cookieValue = (String) testScenario.getInput().get("cookie");
```

Comment:
Remove invalid cookie test cases and remove cookie value from scenarios.
We can read actual valid cookie from constants file for test cases